blocks/simple-list-block/features/simple-list/default.jsx

```diff
@@ -3,12 +3,13 @@ import React from "react";
 import PropTypes from "@arc-fusion/prop-types";
 import { useContent } from "fusion:content";
 import { useFusionContext } from "fusion:context";
-import { LazyLoad, isServerSide } from "@wpmedia/engine-theme-sdk";
-import { extractResizedParams, extractImageFromStory } from "@wpmedia/resizer-image-block";
-import { Heading, HeadingSection } from "@wpmedia/shared-styles";
 import getProperties from "fusion:properties";
+import { LazyLoad } from "@wpmedia/engine-theme-sdk";
+import { extractResizedParams, extractImageFromStory } from "@wpmedia/resizer-image-block";
```

Comment:
these functions can be used from the theme components repo. nearly there! 

if you look at the example from Numbered list pr, that might help 🦉  

https://github.com/WPMedia/arc-themes-blocks/pull/1382/files#diff-699df829c47e7e1bdc79e6e42c2892209a9000ac48f8708cb237c0da73d00ebbR10

Reply:
i think i handled this -- took out the placeholderImage stuff as well (per the numbered list PR).

Reply:
cool nice